"CCMath.h" is missing after installing CCCoreLib

**Describe the bug**
Hey guys since I am new to CloudCompare I am not sure if its really a bug but "CCMath.h" is missing in the `/usr/local/include/CCCorelib` path. Without "CCMath.h" i am not able to use <DistanceComputationTools.h> because the following Error is appearing:  
/usr/local/include/CCCoreLib/Grid3D.h:7:10: fatal error: CCMath.h: No such file or directory
    7 | #include "CCMath.h"

compilation terminated.
ninja: build stopped: subcommand failed.

**How to reproduce**
1. First clone & build & install CCCoreLib
2. Create a CMake and inlcude CCCoreLib

- find_package(CCCoreLib REQUIRED)
- include_directories(${CCCORELIB_INCLUDE_DIR})
- target_link_libraries(${TARGET} PRIVATE CCCoreLib::CCCoreLib)

3. go to your main.cpp and #inlcude <DistanceComputationTools.h>
4. build the project

**Additional context**
Can I just fix the problem for now by copy&paste the "CCMath.h" file to the /usr/local/include/CCCorelib path?

**Your environment**
 - CCCoreLib Version (or git hash/tag): https://github.com/CloudCompare/CCCoreLib.git (cloned on monday 09.08.2021)
 - OS & Version: Ubuntu 21.04
 - Qt Version (if compiling w/Qt): 5.15.2

--- a/include/CMakeLists.txt
+++ b/include/CMakeLists.txt
@@ -71,6 +71,7 @@ install(
 		CCConst.h
 		CCCoreLib.h
 		CCGeom.h
+		CCMath.h
 		CCMiscTools.h
 		CCPlatform.h
 		CCShareable.h

